src/ucp/proto/proto_common.c

```diff
@@ -183,6 +185,14 @@ ucp_proto_common_get_frag_size(const ucp_proto_common_init_params_t *params,
         *max_frag_p = ucs_min(ucp_proto_common_get_seg_size(params, lane),
                               *max_frag_p);
     }
+
+    if (ucs_test_all_flags(params->flags,
+                           UCP_PROTO_COMMON_INIT_FLAG_REMOTE_ACCESS |
+                                   UCP_PROTO_COMMON_INIT_FLAG_SEND_ZCOPY) &&
+        (context->config.ext.rma_zcopy_seg_size != UCS_MEMUNITS_AUTO)) {
+        *max_frag_p = ucs_min(context->config.ext.rma_zcopy_seg_size,
```

Comment:
maybe warn that the requested value was adjustment to minimal allowed one?

Reply:
seems it cause issues with the test. is there any way to disable specific warning in gtest?

Reply:
1. why test does wrong thing?
2. yes, see usage of `scoped_log_handler` in error handling tests

Reply:
in the I modify `rma_zcopy_seg_size` value so now it gets a warning

Reply:
The question is why do you set `rma_zcopy_seg_size ` to unsupported/wrong value which should be adjusted?
> Did you mean to use ucs_warn?

yes, `ucs_warn` or `ucs_diag`